src/routes/+layout.svelte

```diff
@@ -1,7 +1,63 @@
-<script>
+<script lang='ts'>
     import "../app.css";
     import Header from "$lib/components/Header.svelte";
+    import ndk from "$lib/stores/ndk";
+    import currentUser from "$lib/stores/currentUser";
+    import { NDKNip07Signer } from '@nostr-dev-kit/ndk';
+    import { dateTomorrow } from '$lib/utils/helpers';
+    import { goto } from '$app/navigation';
+    import { browser } from '$app/environment';
+    import toast, { Toaster } from 'svelte-french-toast';
+    import Footer from "$lib/components/Footer.svelte";
+
+    let savestore = false;
+
+    $: if (savestore && $currentUser) {
```

Comment:
Happy to chat through all this but it's what does the log in and log out with NDK and a NIP-07 signer.

Reply:
Looks awesome! Thank you for adding all of this in. would love to chat it through in more detail.

I am going to merge, then play around with on my local system.